### PR TITLE
Task/3713 clean up css

### PR DESCRIPTION
**Pull request by tiwariav**
Optimizing css files and removing unused styles.

### CODE REVIEW

**Comment by verysanghee** [comment]
Where are we with this task?

**Comment by tiwariav** [comment]
The optimizations for dcl are done. Have been working on a script to find out and help in frontend related optimizations in future. The script will need more work in future. But I am expecting it to work nicely for similar optimization in hosting and ungleich pages.

**Comment by verysanghee** [comment]
@pcoder tested and looks good so far, could you review?

**Comment by pcoder** [comment]
Also we have some flake8 errors. Please refer CI build output above
```
4.05s$ flake8
./datacenterlight/urls.py:3:1: F401 '.views.SuccessView' imported but unused
./datacenterlight/urls.py:3:1: F401 '.views.LandingProgramView' imported but unused
./datacenterlight/urls.py:3:1: F401 '.views.BetaProgramView' imported but unused
./utils/management/commands/optimize_frontend.py:66:9: F841 local variable 'html_selectors' is assigned to but never used
```

File: utils/management/commands/optimize_frontend.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
+import csv
```
[the remark this answers is not in the document]
Comment (tiwariav) [reply utils/management/commands/optimize_frontend.py]: Yes, definitely. I'll come up with this, once the script is nicely usable.

**Comment by tiwariav** [comment]
@pcoder I've tried documenting the new script. Currently it works only with the `--css` option and can be tested as:
`python manage.py optimize_frontend hosting --css`

I'd propose that we can improve this script further as documented, in a separate PR to help keep the frontend clean.

**Comment by verysanghee** [comment]
Is this good to be merged yet? @tiwariav @pcoder

File: digitalglarus/templates/new_base_glarus.html [hunk 2]
```diff
@@ -72,23 +72,23 @@
...
       }
-	  
-	  
+
+
 	   @media only screen and (max-width: 768px){
 	   .dropdown.home-dropdown-mobile {
           display: block;
-          background-color: 
+          background-color:
```
Comment (pcoder) [inline digitalglarus/templates/new_base_glarus.html]: Missing a value here.

File: datacenterlight/templates/datacenterlight/base_hosting.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        <div class="footer-vm">
+            {% include "datacenterlight/includes/_footer.html" %}
+        </div>
+    {% endif %}
+
+    <!-- Moment -->
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment-with-locales.js"></script>
+    <!-- jQuery -->
+    <script src="{% static 'hosting/js/jquery.js' %}"></script>
```
Comment (pcoder) [inline datacenterlight/templates/datacenterlight/base_hosting.html]: Wonder why we have two separate jquery and bootstrap.min js files for dcl and hosting in the project? If they are the same, we may remove those belonging to hosting.

**Pull request merged**